Observation/Observation-MultipleTumors-Example.json

```diff
@@ -0,0 +1,48 @@
+{
+  "resourceType": "Observation",
+  "id": "Observation-MultipleTumors-Example",
+  "status": "final",
+  "code": {
+    "coding": [
+      {
+        "system": "http://snomed.info/sct",
+        "code": "128462008",
+        "display": "Metastatic malignant neoplasm (disorder)"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/Patient-ZelmaHadjkiss-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307555"
+    }
+  },
+  "specimen": {
+    "identifier": {
+      "system": "https://fhir.add.nhs.uk/Id/specimenId",
+      "value": "RA257630"
+    }
+  },
+  "valueQuantity": {
+    "value": 25,
```

Comment:
Neoplastic cell content does not exist in SNOMED so I would advise splitting this out into a separate observation and changing the code for this to a LOINC one, https://loinc.org/55437-8/